configs/benchmark/templates/pushgateway.yaml

```diff
@@ -32,15 +33,14 @@ metadata:
     visualize: "true"
     app: pushgateway
   name: pushgateway
+  namespace: {{.Release.Namespace}}
```

Comment:
The push gateway really should not be part of the benchmark chart but deployed separately. The push gateway needs to remain available in the cluster after a benchmark concluded, and benchmark resources were deleted.

I'd consider the push gateway a part of the Prometheus component, not the benchmark.

Reply:
I've just uninstalled the benchmark and all the values disappeared from the Grafana dashboard (because the pushgateway was removed I guess). This seems to prevent running multiple benchmarks while retaining metrics if pushgateway is part of the benchmark deployment.

Reply:
Why not simply leave it running?

Reply:
Moved to its own helm chart which deploys in monitoring namespace.